lib/Donors/DonorsContainer.js

```diff
@@ -79,7 +85,8 @@ export function DonorsContainer({
             onAddDonors={onAddDonors}
             name={id}
             searchLabel={searchLabel}
-            visibleColumns={visibleColumns}
+            visibleColumns={pluginVisibleColumnsProp}
```

Comment:
do you really need `pluginVisibleColumnsProp` prop in this component? https://github.com/folio-org/stripes-acq-components/pull/730/files#diff-acbdf62e7810796097a7fc03795e89233de44db880d57ec20583f02a44b8060cR65

Reply:
Yes, I need it. Because Donors form visible columns include `unassignDonor` column name like below
![Screenshot 2023-11-16 at 1 12 42 PM](https://github.com/folio-org/stripes-acq-components/assets/116072773/32d003dc-b8a9-4184-a6ab-647428226ef8)

Reply:
I mean `<DonorsLookup>` has the default `visibleColumns` prop with the same value you are provided as `pluginVisibleColumnsProp`. You don't have to pass `visibleColumns` value to the `<DonorsLookup>` here at all, right?